services/web/server/src/simcore_service_webserver/exporter/formatters/sds/xlsx/templates/code_description.py

```diff
@@ -271,8 +270,8 @@ class CodeDescriptionParams(BaseModel):
     code_description: CodeDescriptionModel = Field(
         ..., description="code description data"
     )
-    inputs: List[InputsEntryModel] = Field([], description="List of inputs, if any")
-    outputs: List[OutputsEntryModel] = Field([], description="List of outputs, if any")
+    inputs: list[InputsEntryModel] = Field([], description="List of inputs, if any")
+    outputs: list[OutputsEntryModel] = Field([], description="List of outputs, if any")
```

Comment:
THOUGH: I am not sure what are the implications with ``[]`` in pydantic vs ``Field(default_factory=list)
It is definitively not a good practice in default arguments of a function

Reply:
Very good point. Changing this one.